content/blog/ggplot2-3-5-0-legends/index.Rmd

```diff
@@ -0,0 +1,544 @@
+---
+output: hugodown::hugo_document
+
+slug: ggplot2-3-5-0-legends
+title: "ggplot2 3.5.0: Legends"
+date: 2024-02-21
+author: Teun van den Brand
+description: >
+    The 3.5.0 version of ggplot2 comes with an overhaul of the guide system.
+    Read here what is new for legends.
+
+photo:
+  url: https://unsplash.com/photos/close-up-photo-of-black-camera-lens-hqCEQTc5gZA
+  author: Markus Spiske
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [ggplot2, ggplot2-3-5-0]
+---
+
+<!--
+TODO:
+* [x] Look over / edit the post's title in the yaml
+* [x] Edit (or delete) the description; note this appears in the Twitter card
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [x] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+
+```{r ragg, include=FALSE}
+knitr::opts_chunk$set(dev = "ragg_png")
+```
+
+We are pleased to release [ggplot2](https://ggplot2.tidyverse.org) 3.5.0.
+This is one blogpost among several outlining changes to legend guides.
+Please find the [main release post](/blog/2024/02/ggplot2-3-5-0/) to read about other changes.
+
+Legends, alongside [axes](/blog/2024/02/ggplot2-3-5-0-axes/), are visual representations of scales and allow observes to translate graphical properties of a plot into information.
+To no surprise, legends in ggplot2 comprise the guides called `guide_legend()`, but also `guide_colourbar()`, `guide_coloursteps()` and `guide_bins()`.
+
+## Styling
+
+One of the more user-visible changes is that these guides no longer have styling options.
+Or at least, they have been soft-deprecated.
```

Comment:
Can you please re-iterate what soft-deprecated means so that people don't get worried that their existing code will break?